roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8710567?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8710567?buildTab=artifacts#/activerecord) on master @ [64a867acd25c0a214209957eefb6483d1158b4f0](https://github.com/cockroachdb/cockroach/commits/64a867acd25c0a214209957eefb6483d1158b4f0):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.2-306-g64a867acd2
Tests run against activerecord 7.0.3
7164 Total Tests Run
7155 tests passed
9 tests failed
25 tests skipped
118 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
/cc @cockroachdb/sql-sessions
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-24551

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -56,6 +56,7 @@ var activeRecordIgnoreList = blocklist{
 	"ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_reference_column_of_child_table":       "flaky - sometimes complains a file is not open for writing",
 	"ActiveRecord::PostgresqlConnectionTest#test_table_exists_logs_name":                                              "flaky - sometimes attempts to call a method on nil",
 	"ActiveRecord::WhereChainTest#test_chaining_multiple":                                                             "flaky - sometimes complains that a relation does not exist",
+	"BasicsTest#test_default_values_are_deeply_dupped":                                                                "flaky - sometimes attempts to call a method on nil",
 	"BinaryTest#test_mixed_encoding":                                                                                                                           "flaky - sometimes complains that a relation does not exist",
 	"CockroachDB::FixturesTest#test_bulk_insert":                                                                                                               "flaky",
 	"CockroachDB::PostgresqlIntervalTest#test_column":                                                                                                          "flaky - sometimes complains that a relation does not exist",

